lbry/error/__init__.py

```diff
@@ -215,9 +215,10 @@ def __init__(self, url):
 
 class ResolveCensoredError(WalletError):
 
-    def __init__(self, url, censor_id):
+    def __init__(self, url, censor_id, censor_row):
         self.url = url
         self.censor_id = censor_id
+        self.censor_row = censor_row
         super().__init__(f"Resolve of '{url}' was censored by channel with claim id '{censor_id}'.")
```

Comment:
This change will get overwritten when the error file is re-generated from the `lbry/error/README.md`.